src/v2/Apps/ViewingRoom/Components/ViewingRoomContentNotAccessible.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { Box, Button, Flex, Sans } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+import { ViewingRoomContentNotAccessible_viewingRoom } from "v2/__generated__/ViewingRoomContentNotAccessible_viewingRoom.graphql"
+
+interface ViewingRoomContentNotAccessibleProps {
+  viewingRoom: ViewingRoomContentNotAccessible_viewingRoom
+}
+
+const ViewingRoomContentNotAccessible: React.FC<ViewingRoomContentNotAccessibleProps> = props => {
```

Comment:
Naming is a bummer. That is as good as I can do. Any better suggestions here?

Reply:
hmmm `ViewingRoomUnavailableContent`? `ViewingRoomInaccessibleStatus`?